### DISCUSSION TITLE
Query for SceneViews with a common Scene

### DISCUSSION THREAD

**Discussion by gbiv** [Vertexvis/vertex-community#21] [Ideas]
After creating a scene, later I'd like to be able to query for the sceneviews that have been created for that scene. Does such an API already exist? 
I wasn't able to find what I was looking for in the API docs of the [Scenes API](https://docs.vertexvis.com/#e5a4dc15-1637-4d15-9567-0c1fde84e0ae) nor the [SceneViews API](https://docs.vertexvis.com/#1874a2bc-9626-43a9-9b25-76d50b13779e).

**Comment by thomas-customer-success** [comment]
@gbiv Thanks for the idea. After some initial investigation this is not something we currently have exposed. However we believe we can be fairly responsive with this request. We have logged a ticket with engineering to see about listing this functionality. Stay tuned.

**Reply by thomas-customer-success** [reply]
@gbiv Update : This API endpoint now exists. It's documented [here](https://docs.vertexvis.com/#c0942ddd-aa49-4a89-a4b5-2856e1fd1b97) and is available in the latest versions of our API clients.

**Reply by gbiv** [reply]
I'll give it a shot
